src/index.html

```diff
@@ -17,16 +17,27 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="input input--large"
+      data-qa="big"
+    >
+      <input
+        class="input__field input__field--large"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
```

Comment:
It's not semantically correct to use a `form` element here. The `form` element should be used when you have user input that will be sent to a server. In this case, you are not sending data anywhere, so it would be more appropriate to use a `div` or other suitable container element.